app/Console/Commands/CheckCommand.php

```diff
@@ -46,6 +47,12 @@ protected function configure(): void
             InputOption::VALUE_IS_ARRAY | InputOption::VALUE_REQUIRED,
             'Class types that should be skipped'
         );
+        $this->addOption(
+            'configuration',
+            'c',
+            InputOption::VALUE_REQUIRED,
+            'Configuration file name'
+        );
```

Comment:
The config should be loaded by default, as convention for these tools.

Reply:
I think it should be done in a separate PR. There are enough conventions to follow (e.g. config files fallback, overriding, schema-based validation...) to make this PR endless.
And as discussed in #8 the goal of this PR is just to provide a "good start".

Reply:
Better solve it now as it's 1-liner here. That would be all to get for merge :+1:

Reply:
But if the default config file does not exist, should the command fail ? Ideally yes, but if so, shouldn't the default config file be distributed alongside the source code ?

Reply:
Like PHPUnit, config should be optional

Reply:
I'm not sure I understand what you mean, if it is just a one-liner could you please add a suggestion ?